src/parser/priest/discipline/modules/core/Channeling.ts

```diff
@@ -8,21 +9,21 @@ const debug = false;
 
 class Channeling extends CoreChanneling {
   _previousPenanceTimestamp = this.owner.fight.start_time - PENANCE_MINIMUM_RECAST_TIME;
-  _hasCastigation = null;
-  _bolt = 0;
+  _hasCastigation: boolean = false;
```

Comment:
Can these (and similar occurrences) not be handled by type inference?

Reply:
I just find that it's cleaner to add the types. They can be inferred if you think I should remove the types. Mind you the disc modules are a bit of a mixed bag for typing modules. There are a ton of nulls that end up being booleans, so I added these in a lot of places to maintain my sanity :D